lib/services.js

```diff
@@ -115,7 +116,7 @@ export default {
         // to return the exact date in the response
         omVal['last-modified'] = lastModifiedDate || new Date().toJSON();
         omVal['content-md5'] = contentMD5;
-
+        omVal['x-amz-replication-status'] = replication;
```

Comment:
I guess this is fine for this PR, but note that I am anticipating that the header `x-amz-replication-status` will not be used, and instead a `replication` property will be added to the object metadata.